lib/models/group.js

```diff
@@ -20,7 +20,7 @@ module.exports.get = function (team, filter) {
       exclude_archived: 1
     }).then(res => (
       res.groups.filter(g => (
-        (g.name.match(filter) || g.purpose.value.match(filter)) &&
+        (g.name.match(filter) || `#${g.name}`.match(filter) || g.purpose.value.match(filter)) &&
```

Comment:
I'd point out that if `g.name.match(filter)` matches, most of the time (without a very strange query) so will ``` `#${g.name}`.match(filter)```; so perhaps we ought to just replace that, rather than supplement?